Eatery Blue/Eatery Blue/UI/HomeScreen/HomeViewController.swift

```diff
@@ -282,11 +283,12 @@ extension HomeViewController: UITableViewDelegate {
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         switch cells[indexPath.row] {
         case .eateryCard(eatery: let eatery):
-            pushViewController(for: eatery)
-
+            allEateries.append(eatery)
```

Comment:
if you append each eatery in `didSelectRowAt`, the order of the eateries in `allEateries` will be in the order you click on them, which will not be the order of the eateries on the screen. also, you will only be able to scroll through the eateries you have previously clicked on, which is not what we want in this case. I would suggest getting `allEateries` from `HomeModelController`, where the data is fetched from backend